src/tvm/node/node.h

```diff
@@ -23,6 +23,12 @@ namespace runtime {
 class NDArray;
 }  // namespace runtime
 
+namespace relay {
+namespace vm {
```

Comment:
perhaps make it runtime::vm::VMObject?

Reply:
I was planning to rename right now, to simply the organization, just pushed a commit.

Reply:
is VMObject a reference class(like NDArray) or the container itself? Ideally we want it to be the reference class

Reply:
could likely be merged into the namspace above(runtime::NDArray)

Reply:
It is a reference class, we can move into runtime if you want

Reply:
You can make the decison. Perhaps it makes sense to move to runtime given that there is no other runtime Object

Reply:
yeah, not sure if we should keep it isolated or not. Maybe it makes sense if we will reuse for AoT/JiT, etc.